src/index.js

```diff
@@ -0,0 +1,181 @@
+"use strict";
+
+// GLOBAL VARIABLES
+const increaseTempControl = document.getElementById('increaseTempControl');
+const decreaseTempControl = document.getElementById('decreaseTempControl');
+const tempValue = document.getElementById('tempValue');
+const landscape = document.getElementById('landscape');
+const currentTempButton = document.getElementById('currentTempButton');
+const cityNameInput = document.getElementById('cityNameInput');
+const headerCityName = document.getElementById('headerCityName');
+const skySelect = document.getElementById('skySelect');
+const sky = document.getElementById('sky')
+const gardenSection = document.getElementById('gardenSection');
+const cityNameReset = document.getElementById('cityNameReset');
+
+
+const defaultTemp = () => {
+  const defaultTemperature = 70;
+  tempValue.textContent = defaultTemperature;
+  updateTempColor(defaultTemperature);
+  updateLandscape();
+}
+
+// WAVE 2
+const updateTemp = (increment) => {
+  let currentTemp = parseInt(tempValue.textContent);
+  let newTemp = currentTemp + increment;
+
+  tempValue.textContent = newTemp;
+
+  updateTempColor(newTemp);
+}
+
+const updateTempColor = (temperature) => {
+  tempValue.classList.remove('red','orange','yellow','green','turqoise');
+
+  if (temperature > 79) {
+    tempValue.classList.add('red');
+  } else if (temperature > 69) {
+    tempValue.classList.add('orange');
+  } else if (temperature > 59) {
+    tempValue.classList.add('yellow');
+  } else if (temperature > 49) {
+    tempValue.classList.add('green');
+  } else {
+    tempValue.classList.add('turqoise');
+  }
+
+  updateLandscape();
+}
+
+const updateLandscape = () => {
+  let currentTemp = parseInt(tempValue.textContent);
+
+  if (currentTemp >= 79) {
+    landscape.textContent = '🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂';
+  } else if (currentTemp >= 69) {
+    landscape.textContent = '🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷';
+  } else if (currentTemp >= 59) {
+    landscape.textContent = '🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃';
+  } else {
+    landscape.textContent = '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲';
+  }
+}
+
+// WAVE 3
+const updateCityName = () => {
+  cityNameInput.addEventListener("input", updateValue);
+
+  function updateValue(e) {
+    headerCityName.textContent = e.target.value;
+  }
+}
+
+// WAVE 4
+const findLatitudeAndLongitude = async (query) => {
+  try {
+    const response = await axios
+      .get('http://127.0.0.1:5000/location', {
+        params: {
+          q: query,
+          format: 'json'
+        }
+      });
+
+    const latitude = response.data[0].lat;
+    const longitude = response.data[0].lon;
+    console.log('success in findLatitudeAndLongitude!', 'latitude:', latitude, 'longitude:', longitude);
+
+    return await findWeather(latitude, longitude);
```

Comment:
This function is not returning what its name suggests. It's called `findLatitudeAndLongitude` but instead of returning the latitude and longitude and letting the original caller do something with those values, the function feeds the latitude and longitude into another function `findWeather` and returns whatever that function gives back. 

This is a pattern that is likely to cause confusion for folks reading through the code or working in the project. These two functions are now tightly coupled and we lose some of the benefit of having them separated out into their own named functions. 
- How could we restructure how we use these two functions to make them better hold to the single responsibility principle and have the actions they take better reflect their names?